core/src/apps/management/backup_device.py

```diff
@@ -10,17 +10,26 @@
 
 
 async def backup_device(msg: BackupDevice) -> Success:
+    import storage.cache as storage_cache
     import storage.device as storage_device
     from trezor import wire
     from trezor.messages import Success
 
+    from apps import workflow_handlers
     from apps.common import mnemonic
 
     from .reset_device import backup_seed, backup_slip39_custom, layout
 
+    # do this early before we show any UI
+    # the homescreen will clear the flag right after its own UI is gone
+    repeated_backup_unlocked = (
+        storage_cache.get(storage_cache.APP_RECOVERY_REPEATED_BACKUP_UNLOCKED)
+        == b"\x01"
```

Comment:
A zero-length string would work, but it would be rather confusing. A constant seems much better.

Reply:
actually the style we use with flags is setting <anything> for True and `b""` for False -- so that you can do `if cache.get(XYZ)` without comparing to a particular value, and the unset version comes out falsey.

i don't think `b"\x01"` is magic enough to warrant a named constant taking up space and being imported in multiple places

Reply:
I'd generally prefer something more explicit, like `storage_cache.set_bool()`, but let's at least get rid of the `== b"\x01"` when testing the values and do what @matejcik described.

Reply:
I actually did `set_bool` / `get_bool` after all and I like how it looks. We can now always change what these functions set behind the scenes if we wish to.

Reply:
https://github.com/trezor/trezor-firmware/pull/3743/files#diff-c6b39a04397efa8d6140a9d7d73c86e40430c5c6fab2b65d61cbfd1627e7e0dcR258